src/style.css

```diff
@@ -1,3 +1,87 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap';
+
+:root {
+  --active-link-color: #00acdc;
+}
+
+a {
+  display: inline-block;
+  padding: 0;
+  margin: 0;
+  line-height: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 6.5px 50px;
+}
+
+.header__icon {
+  width: 40px;
+  height: 40px;
+}
+
+.header__icon img {
+  display: block;
+}
+
+.header__list {
+  display: flex;
+  align-items: center;
+  font-weight: 500;
+  height: 15px;
+}
+
+.header__list_item {
+  list-style: none;
```

Comment:
Avoid using tag names for styling elements. Use class selectors instead to apply styles to your list items.